app/src/main/java/com/github/se/icebreakrr/ui/sections/Notifications.kt

```diff
@@ -73,19 +77,19 @@ fun NotificationScreen(navigationActions: NavigationActions, profileViewModel: P
                         Modifier.padding(vertical = TEXT_VERTICAL_PADDING)
                             .testTag("notificationFirstText"))
                 Column(verticalArrangement = Arrangement.spacedBy(CARD_SPACING)) {
-                  cardList.value.take(MAX_PENDING_CARDS).forEach { p ->
-                    ProfileCard(p, onclick = navFunction)
-                  }
-                }
-                Text(
-                    text = PASSED,
-                    fontWeight = FontWeight.Bold,
-                    modifier =
-                        Modifier.padding(vertical = TEXT_VERTICAL_PADDING)
-                            .testTag("notificationSecondText"))
-                Column(verticalArrangement = Arrangement.spacedBy(CARD_SPACING)) {
-                  cardList.value.drop(MAX_PENDING_CARDS).forEach { p ->
-                    ProfileCard(p, onclick = navFunction, greyedOut = true)
+                  cardList.value.forEach { p ->
+                    ProfileCard(
+                        p.key,
+                        onclick = {
+                          if (p.key.fcmToken != null) {
+                            meetingRequestViewModel.setMeetingResponse(
+                                p.key.fcmToken!!, MEETING_REQUEST_ACCEPTED, true)
+                            meetingRequestViewModel.sendMeetingResponse()
+                            meetingRequestViewModel.removeFromMeetingRequestInbox(p.key.uid)
+                            profileViewModel.getSelfProfile()
+                            profileViewModel.getInboxOfSelfProfile()
```

Comment:
You can add error management here if the `fcmToken` is `null` so that the application doesn't crash if this happens.

Reply:
Yes i ll add an error log. But normaly it is impossible that the token is null since for all users, the fcm token is set up at SignIn so the error would have appeared before